tools/log_rotate/install_script.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Create directory, which keeps all logrotate-config to rotate hourly.
+mkdir /etc/logrotate.hourly.d
+
+cat << EOF | tee /etc/logrotate.hourly.conf
+# packages drop hourly log rotation information into this directory
+include /etc/logrotate.hourly.d
+EOF
+
+chmod 644 /etc/logrotate.hourly.conf
+
+cat << EOF | tee /etc/cron.hourly/logrotate
+#!/bin/bash
```

Comment:
similar to above comment, assuming this will not interfere with existing configurations in logrotate file (if there are any)

Reply:
Yes.